src/server/rpc/procedures/british-museum/british-museum.js

```diff
@@ -131,6 +143,12 @@ britishmuseum.searchByType = function(type, limit){
 };
 
 
+/**
+ * Search for artifacts by material
+ * @param {String} material Material to search for
+ * @param {Number=} limit Maximum number of artifacts to return
```

Comment:
Technically, this should be a bounded number greater than (or debatably equal to) 0. Setting it to zero will fall back on the default but negative numbers will cause an error from the BritishMuseum endpoint (`500 - ""`).